Communitter/src/main/java/com/communitter/api/util/ImageUtils.java

```
@@ -0,0 +1,47 @@
+package com.communitter.api.util;
+
+import java.io.ByteArrayOutputStream;
+import java.util.zip.Deflater;
+import java.util.zip.Inflater;
+
+public class ImageUtils {
+
+
+    public static byte[] compressImage(byte[] data) {
+
+        Deflater deflater = new Deflater();
+        deflater.setLevel(Deflater.BEST_COMPRESSION);
+        deflater.setInput(data);
+        deflater.finish();
+
+        ByteArrayOutputStream outputStream = new ByteArrayOutputStream(data.length);
+        byte[] tmp = new byte[4*1024];
+        while (!deflater.finished()) {
+            int size = deflater.deflate(tmp);
+            outputStream.write(tmp, 0, size);
+        }
+        try {
```

if it is necessary we should throw the error